flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PartitionTestUtils.java

```diff
@@ -58,6 +58,21 @@ public static ResultPartition createPartition(ResultPartitionType type, FileChan
 			.build();
 	}
 
+	public static ResultPartition createPartition(
+			ResultPartitionType type,
+			FileChannelManager channelManager,
+			boolean compressEnabled,
+			String  compressCodec,
```

Comment:
This parameter might be not needed because we always have only one codec for compression.